halium-install

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This doesn't work with /bin/sh, so I switched to bash. If you know any reason why we should only use the sh, please report :)